edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -91,32 +96,33 @@ public class KubernetesModule : Module
         protected override void Load(ContainerBuilder builder)
         {
             // IKubernetesClient
-            builder.Register(c =>
-            {
-                if (this.enableServiceCallTracing)
-                {
-                    // enable tracing of k8s requests made by the client
-                    var loggerFactory = c.Resolve<ILoggerFactory>();
-                    ILogger logger = loggerFactory.CreateLogger(typeof(Kubernetes));
-                    ServiceClientTracing.IsEnabled = true;
-                    ServiceClientTracing.AddTracingInterceptor(new DebugTracer(logger));
-                }
+            builder.Register(
+                    c =>
```

Comment:
Why did you move these down to their own line? It increases the amount of nesting. Can you put these back?

Reply:
Same comment for the instances in this file.

Reply:
Probably its IDE formatting that I need to configure